recipes/cudatoolkit/build.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+set -ex
+
+echo "Building cudatoolkit ..."
+
+filename="cuda_${PKG_VERSION}"
+install_dir=$CONDA_PREFIX/tmp/cuda_install
+tmp_dir=$CONDA_PREFIX/tmp/cuda_tmp
+
+echo "Creating folders"
+mkdir -p $install_dir $tmp_dir
+mkdir -p $PREFIX/{lib,include}
+
+# Install
+UNAME=`uname`
+if [[ $UNAME == "Linux" ]]; then
+    find_args=""
+    libs_path=""
+    chmod ugo+x $filename
+    ./$filename --silent --toolkit --toolkitpath=$install_dir --override
+else
+    find_args="-type f"
```

Comment:
I think you want `-xtype f` here instead, so it doesn't pick up symbolic links.

Reply:
@scopatz thanks .. I will try that ... In our server it is already working ...

Reply:
the output of the building on our server:

```
===== testing package: cudatoolkit-9.2-0 =====
running run_test.py
Finding cublas
        located at /Users/Shared/xmnlab/conda-forge/miniconda3/conda-bld/cudatoolkit_1533086615862/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/libcublas.9.2.dylib
        trying to open library...       ok
Finding cusparse
        located at /Users/Shared/xmnlab/conda-forge/miniconda3/conda-bld/cudatoolkit_1533086615862/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/libcusparse.9.2.dylib
        trying to open library...       ok
Finding cufft
        located at /Users/Shared/xmnlab/conda-forge/miniconda3/conda-bld/cudatoolkit_1533086615862/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/libcufft.9.2.dylib
        trying to open library...       ok
Finding curand
        located at /Users/Shared/xmnlab/conda-forge/miniconda3/conda-bld/cudatoolkit_1533086615862/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/libcurand.9.2.dylib
        trying to open library...       ok
Finding nvvm
        located at /Users/Shared/xmnlab/conda-forge/miniconda3/conda-bld/cudatoolkit_1533086615862/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/libnvvm.3.dylib
        trying to open library...       ok
        finding libdevice for compute_20...     ok
        finding libdevice for compute_30...     ok
        finding libdevice for compute_35...     ok
        finding libdevice for compute_50...     ok
NVVM version (1, 5)

Resource usage statistics from testing cudatoolkit:
   Process count: 2
   CPU time: Sys=0:00:00.3, User=0:00:00.4
   Memory: 47.8M
   Disk usage: 24B
   Time elapsed: 0:00:04.2

TEST END: /Users/Shared/xmnlab/conda-forge/miniconda3/conda-bld/osx-64/cudatoolkit-9.2-0.tar.bz2
```

Reply:
@scopatz it seems xtype is not known: `find: -xtype: unknown primary or operator`

this is the info about the -type parameter:

``` 
 -type t
             True if the file is of the specified type.  Possible file types are as follows:

             b       block special
             c       character special
             d       directory
             f       regular file
             l       symbolic link
             p       FIFO
             s       socket

```